lxd/patches.go

```diff
@@ -345,7 +347,15 @@ func upgradeFromStorageTypeBtrfs(name string, d *Daemon, defaultPoolName string,
 		if shared.PathExists(oldContainerMntPoint) {
 			err = os.Rename(oldContainerMntPoint, newContainerMntPoint)
 			if err != nil {
-				return err
+				err = btrfsSnapshot(oldContainerMntPoint, newContainerMntPoint, false)
+				if err != nil {
+					return err
+				}
+				err = btrfsSubVolumesDelete(oldContainerMntPoint)
+				if err != nil {
+					btrfsSubVolumesDelete(newContainerMntPoint)
+					return err
+				}
```

Comment:
Hm, ah snapshots... So we could try to do a btrfssubvolumecreate and rsync over
but it might not be worth the trouble